orga/app/routes/authenticated/sup-students/import.js

```diff
@@ -1,3 +1,13 @@
+import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
 
-export default class ImportRoute extends Route {}
+export default class ImportRoute extends Route {
+  @service currentUser;
+
+  beforeModel() {
+    super.beforeModel(...arguments);
+    if (!this.currentUser.isAdminInOrganization) {
+      return this.replaceWith('application');
+    }
+  }
```

Comment:
[question] Est-ce que ce type de check ne devrait pas être dans le hook `redirect` ?

Reply:
Il me semble que `redirect` est exécuté après `beforeModel`, `model` et `afterModel`, et que la convention Ember quand on a pas les droits, c'est de rediriger dans le `beforeModel`, vu qu'on est avant d'avoir véritablement rejoins la route.

Reply:
la doc dit : "Since a route's beforeModel() executes before the model() hook, it's a good place to do a redirect if you don't need any information that is contained in the model."